main.ts

```diff
@@ -1,30 +1,42 @@
 class Student {
-    Id:number;
-    Name:string;
-    Age:number;
-    Gender?:string;
-    Address?:string;
-    AdmissionMonth?:string;
-    Course?:string[];
+    id : number;
+    name : string;
+    age : number;
+    gender? : string;
+    address? : string;
+    admissionMonth? : string;
+    course? : string;
+
+    constructor(
+        id : number,
+        name : string,
+        age : number,
+        gender? : string,
+        address? : string,
+        admissionMonth? : string,
+        course? : string,
+    ){
+        this.id = id;
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.address = address;
+        this.admissionMonth = admissionMonth;
+        this.course = course;
+
+    };
 }
 
 
 class School {
-    student1 = new Student();
-    student2 = new Student();
-
-    student1.Age = 10;
-    student2.Age = 20;
-
-    let studentAge1 = student1.Age;
-    let studentAge2 = student2.Age;
-
-    ageSum(studentAge1,studentAge2){
-
-        return studentAge1+studentAge2;
+    studentAgeSum(studentAge1: number , studentAge2: number){
+        return (studentAge1 + studentAge2);
     }
 
 }
+let student1 = new Student( 1,'None',15,'None','None','None','None');
```

Comment:
**No space** at the start and end of brackets. So fix the space before `1`. Also add a `space` after `,`.